src/ng/compile.js

```diff
@@ -3317,11 +3428,10 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
     }
 
 
-    function getTrustedContext(node, attrNormalizedName) {
+    function getTrustedAttrContext(tag, attrNormalizedName) {
```

Comment:
Actually this function previously had `var tag = nodeName_(node)`, so changing this will increase the diff. Still want to change it for the consistency?

Reply:
I'd take larger diff over consistency any day. (But I admit that's a personal preference :grin:)
Up to you.

Reply:
Done: d1264cde1